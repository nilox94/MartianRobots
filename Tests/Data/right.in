1 1
0 0 N
R
0 0 E
R
0 0 S
R
0 0 W
R
