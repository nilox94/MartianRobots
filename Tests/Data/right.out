0 0 E
0 0 S
0 0 W
0 0 N
